l3kernel/l3str.dtx

```diff
@@ -106,7 +106,7 @@
 %   initially empty.
 % \end{function}
 %
-% \begin{function}[added = 2015-09-18, updated = 2018-07-28]
+% \begin{function}[added = 2015-09-18, updated = 2023-11-25]
```

Comment:
Nothing has changed in the API, so these are not updated

Reply:
reverted